### PR TITLE
add video welcome screen onboarding and desyncoverlay

### PR DESCRIPTION
**Pull request by pierrelouis-c**
### 📝 Description

Add video to the welcome screen on the onboarding

### ❓ Context

- **Impacted projects**: `ledger-live-desktop`
- **Linked resource(s)**: `[FAT-219]`

### ✅ Checklist

- [ ] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 📸 Demo

videoWelcomeOnboarding

overlayDO

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[FAT-219]: https://ledgerhq.atlassian.net/browse/FAT-219?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Welcome/index.tsx [hunk 1]
```diff
@@ -79,17 +70,21 @@ const CarouselTopBar = styled(Flex).attrs({
   alignItems: "center",
   padding: "40px",
   width: "100%",
+  zIndex: "1",
```
Comment (thomasrogerlux) [inline apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Welcome/index.tsx:73]: Why is it a string? Shouldn't it be a number?

**Review by thomasrogerlux — APPROVED** [review]

File: apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/DesyncOverlay.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        </Box>
+      </Flex>
+      <Flex position="absolute" width="100%" justifyContent="flex-end" bottom={0} padding={4}>
+        <Flex
+          width="400px"
+          backgroundColor={colors.warning}
+          borderRadius="8px"
+          p={4}
+          mr="24px"
+          mb="24px"
```
Comment (thomasrogerlux) [inline apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/DesyncOverlay.tsx:56]: Use `mr={8}` and `mb={8}`

File: apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/DesyncOverlay.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      position="absolute"
+      backgroundColor={colors.constant.overlay}
+      top={0}
+      left={0}
+      height="100%"
+      width="100%"
+      flexDirection="column"
+    >
+      <Flex justifyContent="flex-end" mr="24px" mt="24px">
+        <Box padding="4px 8px 7px 8px" borderRadius="320px" backgroundColor={colors.neutral.c30}>
```
Comment (thomasrogerlux) [inline apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/DesyncOverlay.tsx:45]: Remove the Box use the button only but wait for my PR to rebase

**Review by thomasrogerlux — APPROVED** [review]

**Comment by gre** [comment]
any way to cover the tests?

**Comment by thomasrogerlux** [comment]
> any way to cover the tests?

We're gonna write tests with Playwright but directly in the sync onboarding branch, we have a task to setup the model and all and start testing the new flow

**Review by thomasrogerlux — APPROVED** [review]

**Comment by thomasrogerlux** [comment]
Add changeset before merging too

**Pull request merged**